app/assets/stylesheets/orchid.scss

```diff
@@ -34,6 +34,10 @@
 // Site Section-specific
 // =====================
 .site_search {
+  .pagination_container {
```

Comment:
The property should be: `text-align: right;` Space between and ending with a semi-colon.

Also use Sass nesting:
```css
.pagination_container {
  text-align: right;

  .pagination {
    margin: 0;
  }
}
```

Reply:
Do these really need to be nested?

Reply:
They don't _need_ to be but I prefer it to DRY up naming the parent class. Plus I like how it mimics the parent-child relationship of the elements being styled. It's visually more like how the markup is nested/indented.

Reply:
it has been done